integration-tests/submitProposal.ts

```diff
@@ -12,7 +12,7 @@ const client = new LCDClient({
   URL: 'https://bombay-lcd.terra.dev',
 });
 
-// LocalTerra test1 terra1x46rqay4d3cssq8gxxvqz8xt6nwlz4td20k38v
+// LocalPaloma test1 terra1x46rqay4d3cssq8gxxvqz8xt6nwlz4td20k38v
```

Comment:
Another Terra address

Reply:
... and above there is actually a reference to bombay-lcd.terra.dev 

This needs more care than just a simple find and replace